cloud/deploys/bin/azure/db-connection

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env bash
+
+# DOC: Connect to the database via the bastion
+# example: ./db-connection -g sgdev -d civiform-artistic-halibut
+key_name="bastion"
+while getopts g:d:k flag
+do
+    case "${flag}" in
+        g) resource_group=${OPTARG};;
+        d) database_name=${OPTARG};;
+        k) key_name=${OPTARG};;
+    esac
+done
+
+if [[ ! -z "${resource_group}" ]] || [[ ! -z "${database_name}" ]]; then
+  echo "arguments -g and -d must be provided"
```

Comment:
suggestion (blocking): print to stderr for error messages

Reply:
done!